parts/k8s/containeraddons/1.16/kubernetesmasteraddons-omsagent-daemonset.yaml

```diff
@@ -67,79 +67,58 @@ data:
      type forward
      port "#{ENV['HEALTHMODEL_REPLICASET_SERVICE_SERVICE_PORT']}"
      bind 0.0.0.0
+     chunk_size_limit 4m
     </source>
      #Kubernetes pod inventory
      <source>
       type kubepodinventory
       tag oms.containerinsights.KubePodInventory
-      run_interval 60s
+      run_interval 60
       log_level debug
      </source>
 
      #Kubernetes events
      <source>
       type kubeevents
       tag oms.containerinsights.KubeEvents
-      run_interval 60s
+      run_interval 60
       log_level debug
       </source>
 
-     #Kubernetes logs
-     <source>
-      type kubelogs
-      tag oms.api.KubeLogs
-      run_interval 60s
-     </source>
-
-     #Kubernetes services
-     <source>
-      type kubeservices
-      tag oms.containerinsights.KubeServices
-      run_interval 60s
-      log_level debug
-     </source>
-
      #Kubernetes Nodes
      <source>
       type kubenodeinventory
       tag oms.containerinsights.KubeNodeInventory
-      run_interval 60s
-      log_level debug
-     </source>
-
-     #Kubernetes perf
-     <source>
-      type kubeperf
-      tag oms.api.KubePerf
-      run_interval 60s
+      run_interval 60
       log_level debug
      </source>
 
     #Kubernetes health
     <source>
      type kubehealth
      tag kubehealth.ReplicaSet
-     run_interval 60s
+     run_interval 60
```

Comment:
is this intentional removing units (i.e. s) for this field here?

Reply:
yes. This is related to scheduler optimization

Reply:
> yes. This is related to scheduler optimization

thanks.